### ISSUE TITLE
BuildPhase "none" not possible for folders

### ISSUE THREAD

**Issue by Lutzifer** [yonaskolb/XcodeGen#287]
Adding a folder without copying it (so having no Target Membership) does not work:

```
- path: Carthage/Checkouts
   buildPhase: none
   type: folder
```

setting buildPhase to headers seems to have the effect, but it seems wrong 😉

**Comment by yonaskolb** [comment]
Thanks for raising this @Lutzifer. Fixed in #288.
Is `resources` still a good default for folders I wonder.

What's the motivation for adding the Carthage checkouts as a folder? They get added within `Frameworks/Carthage` automatically if you use any of them as a dependency

**Comment by Lutzifer** [comment]
Thanks for the fix.

One of our engineers recently discovered (based loosely on https://github.com/Carthage/Carthage/issues/2137) that adding the checkout folder this way improves Xcode's debugging abilities when it comes to the Carthage Frameworks.
